.changelog/add-change.sh

```
@@ -0,0 +1,377 @@
+                printf 'Unknown argument: [%s].\n' "$1"
+                printf 'The <num> can only contain digits.\n'
+                printf 'The <id> and <section> can only contain alphanumeric characters and dashes.\n'
+                printf 'The <message> must contain at least one space.\n'
+                exit 1
+            fi
+            ;;
+    esac
+    shift
+done
+
+where_i_am="$( cd "$( dirname "${BASH_SOURCE:-$0}" )"; pwd -P )"
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)